app/models/changeset.rb

```diff
@@ -84,11 +70,33 @@ def error
 
   private
 
+  def find_comparison
+    if empty?
+      NullComparison.new('Start and end commits are the same')
```

Comment:
I would not consider this an error / would not want to see the warning UI, maybe pass nil